README.md

```diff
@@ -39,7 +36,7 @@ To do this follow the instructions in the
 
 Now that Qiskit Machine Learning is installed, it's time to begin working with the Machine Learning module.
 Let's try an experiment using VQC (Variational Quantum Classifier) algorithm to
-train and test samples from a data set to see how accurately the test set can
+train and test samples from a dataset to see how accurately the test set can
```

Comment:
I think either 'data set' or 'dataset' is fine. However since we use dataset earlier for consistency I guess this works better.